view/loadView.js

```diff
@@ -306,9 +306,9 @@ module.exports = function loadView(app, passport) {
 
         // return token
         const token = jwtUtil.createToken(req.user.name, req.user.name);
+        req.session.token = token;
```

Comment:
I guess we can do this without any problem. Can I try to remove that ? @pallavi2209 @iamigo